src/Test/Utils/custom-queries.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  queryHelpers,
+  buildQueries,
+  Matcher,
+  MatcherOptions,
+} from "@testing-library/react";
+
+const getMultipleError = (attrName: string) => (_c, selectorValue) =>
```

Comment:
aren't these supposed to be in the "words" file?

Reply:
The composer is part of the app, the words file is a dictionary of any pieces of text (or at least should, there are places where it wasn't used out of laziness I assume) :D So you can add those two in the file :D

Reply:
I wouldn’t mix error messages from tests with displayed text, but I don't mind, I'll add those :D